src/transaction/log_impl.h

```diff
@@ -759,6 +760,18 @@ typedef struct log_logging_stat
   unsigned long async_commit_request_count;
 } LOG_LOGGING_STAT;
 
+/*for CDC interface */
+typedef struct cdc_server_comm
+{
+  char *log_Infos;
+  int log_Info_length;
+  int num_log_Infos;
+  LOG_LSA start_lsa;
+  bool is_sent;
+} CDC_SERVER_COMM;
+
+extern CDC_SERVER_COMM server_comm_buf;
```

Comment:
Follow the global variable name convention.
```suggestion
extern CDC_SERVER_COMM cdc_server_Comm_buf;
```